docs/csharp/programming-guide/classes-and-structs/access-modifiers.md

```diff
@@ -18,6 +18,17 @@ All types and type members have an accessibility level. The accessibility level
 - [protected internal](../../language-reference/keywords/protected-internal.md): The type or member can be accessed by any code in the assembly in which it's declared, or from within a derived `class` in another assembly.
 - [private protected](../../language-reference/keywords/private-protected.md): The type or member can be accessed only within its declaring assembly, by code in the same `class` or in a type that is derived from that `class`.
 
+**Summary Table**
+| Caller's location ↓  | public | protected internal | protected | internal | private protected | private 
```

Comment:
OK, I missed this one as well. We need the final closing `|` on the table lines:

(Have I mentioned I hate tables in markdown 😆 )

```suggestion
| Caller's location ↓  | public | protected internal | protected | internal | private protected | private |
```

Reply:
Just the first row?
(I start to see why I should hate tables in markdown as well 😆)

Reply:
It should be all rows.

Reply:
Yeah, that's what I thought, just wanted to be sure.